11_styling_with_tailwind/src/components/MemberDetails.js

```diff
@@ -0,0 +1,102 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
+import company_logo from "../assets/company.png";
+import location_logo from "../assets/location.png";
+
+const Member = () => {
+	let { username } = useParams();
+	const [member, setMember] = useState("");
+
+	if (!username) return;
+
+	useEffect(() => {
+		getMemberDetails();
+	}, []);
+
+	const getMemberDetails = async () => {
+		const usersData = JSON.parse(sessionStorage.getItem("usersData") || []);
+		let user;
+
+		// check sessionStorage, else get data from github API
+		if (usersData) {
+			user = usersData.find((entry) => entry.login == username);
+		} else {
+			const userInfo = await fetch(
+				`https://api.github.com/users/${username}`
+			);
+			user = await userInfo.json();
+		}
+
+		setMember(user);
+	};
+
+	if (!member) return;
+
+	const {
+		avatar_url,
+		bio,
+		name,
+		login,
+		followers,
+		following,
+		location,
+		company,
+	} = member;
+
+	return (
+		<div className="member-info items-left flex flex-col justify-center p-24">
```

Comment:
why is member-infp present in classname when using tailwind css?

Reply:
These were older class names where styles were applied. Removed